hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,3 @@
+RELEASE_TYPE: patch
+
+This patch improves our shrinking of :func:`~hypothesis.strategies.sets` and :func:`~hypothesis.strategies.lists` with ``unique=True`` (:pull:`3987`).
```

Comment:
Also dictionaries!  And anywhere else that our internals rest on unique lists, which is _quite a few places_ 😁

Reply:
please feel free to rewrite this changelog, or drop a string here for me to commit! You know better than I what is impacted here and how to convey it, and imo changelog messages are pretty low-effort / high-impact in terms of their reach, so it's good to get them right.